packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -717,6 +720,15 @@ export function fromTask(task: theia.Task): TaskDto | undefined {
         taskDto.scope = task.scope;
     }
 
+    const group = task.group;
+    if (group) {
```

Comment:
minor: here and below you check `if (group)` before checking `if (group === something)`. I think you could skip the outer check, since `if (!group)`, definitely `!(group === something)`. Otherwise, this PR looks good to me

Reply:
Thanks for pointing it out Colin, 
It's now fixed in a new version of the commit.